package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
If the newer version of minimatch is compatible with the mocha etc we can use npm shrinkwrap to specify which version of minimatch is installed in dependency node_modules